### PR TITLE
Rename ovm and vm

### PR DESCRIPTION
**Pull request by SchSeba**
Change the get start with kubevirt page.

rename ovms to vms and vms to vmis

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt.github.io/56)

### CODE REVIEW

**Comment by scollier** [comment]
LGTM

File: pages/get_kubevirt.md [hunk 1]
```diff
@@ -59,19 +59,19 @@ Once you deployed KubeVirt you are ready to launch a VM:
 $ kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml
 
 # After deployment you can manage VMs using the usual verbs:
-$ kubectl get ovms
-$ kubectl get ovms -o yaml testvm
+$ kubectl get vms
+$ kubectl get vms -o yaml testvm
```
Comment (mhenriks) [inline pages/get_kubevirt.md]: If I run `kubectl get vms` before the vm is started (the order in this snippet) I get the following error:  `No resources found.`

File: pages/get_kubevirt.md [hunk 2]
```diff
@@ -59,19 +59,19 @@ Once you deployed KubeVirt you are ready to launch a VM:
...
 
 # To start an offline VM you can use
 $ ./virtctl start testvm
-$ kubectl get vms
-$ kubectl get vms -o yaml testvm
+$ kubectl get vmis
+$ kubectl get vmis -o yaml testvm
```
Comment (mhenriks) [inline pages/get_kubevirt.md]: Running `kubectl get vmis` at this point in the snippet gives me this error:  `error: the server doesn't have a resource type "vmis"`

I also don't see 'vmis' defined anywhere in the yaml file that installs kubevirt.

File: pages/get_kubevirt.md [hunk 3]
```diff
@@ -59,19 +59,19 @@ Once you deployed KubeVirt you are ready to launch a VM:
...
 
 # To shut it down again
 $ ./virtctl stop testvm
 
 # To delete
-$ kubectl delete vms testvm
+$ kubectl delete vmis testvm
```
Comment (mhenriks) [inline pages/get_kubevirt.md]: This does not work either.  But neither did `kubectl delete vms testvm`.  It appears that command can only be run when the vm is started.  Not after it's stopped.  I think `kubectl delete ovms testvm` is what we want here.

File: pages/get_kubevirt.md (hunk 1 above)
Comment (SchSeba) [reply pages/get_kubevirt.md]: Yes this release was a preparation for the v0.7.0 release with the rename of the offlinevirtualmachine to virtualmachine and virtualmachine to virtualmachineinstace

**Comment by karmab** [comment]
addressed by https://github.com/kubevirt/kubevirt.github.io/pull/58
as @slintes pointed out, the version also needs to be updated to a one (>0.7.0*) supporting vm/vmi

**Comment by SchSeba** [comment]
Yes I know, but I am not sure we want to point an alpha release I think its better to revert this until we release v0.7.0

**Pull request merged**